scripts/cancelBatch.py

```diff
@@ -0,0 +1,53 @@
+# Script that takes in an array of batch names and applies take a bulk action such as removing the unique id or canceling all tasks.
+# By default, this script makes 50 concurrent API calls.
+import scaleapi
+from scaleapi.exceptions import ScaleException
+
+import argparse
+from concurrent.futures import ThreadPoolExecutor
+
+# Change this for update concurrency
+MAX_WORKERS = 50 
+
+# globals
+task_count = 0
+task_errors = []
+tasks_cancelled = []
+
+def main():
+
+    args = get_args()
+    clearUniqueId = args.clear or False
+
+    client = scaleapi.ScaleClient(args.api_key)
+    preProcessedBatches = args.batches.split(",")
+    batches = [word.strip() for word in preProcessedBatches]
+    for batch in batches:
+
+        task_count = client.get_tasks_count(batch_name = batch)
+        print("Batch ", batch, " contains ", task_count, " tasks.")
+        
+        tasks = client.get_tasks(batch_name = batch)
+        with ThreadPoolExecutor(max_workers=MAX_WORKERS) as executor:
+            for task in tasks:
+                try:
+                    tasks_cancelled.append(executor.submit(client.cancel_task(task.task_id, clearUniqueId)))
+                except ScaleException as err:
+                    task_errors.append({
+                        "task": task.task_id,
+                        "message": err.message,
+                        "code": err.code
+                    })
+    
+    print("Tasks Canceled", len(tasks_cancelled))
+    print("Tasks Errored", len(task_errors))
+
+def get_args():
+    ap = argparse.ArgumentParser()
+    ap.add_argument('--api_key', required=True)
```

Comment:
We can add tips/comments to command line arg parameters. In example:
```suggestion
    ap.add_argument('--api_key', required=True, help="Please provide Scale API Key")
```